doc/manuális_tesztelés.md

```diff
@@ -0,0 +1,145 @@
+# Manuális tesztek
+
+## Karakterkezelés teszt
+
+Az input .csv fájlban ékezetes (á, é, ő, ű) betűk szerepelnek, a teszt célja megvizsgálni, hogy az alkalmazás ezeket megfelelően kezeli és az outputban is helyesen szerepelnek.
+
+Az input file: `karakterkezeles.csv`
```

Comment:
csak ötlet, a fájlokra akár mutathatna link is, ha itt így megjelölitek
pl: [karakterkezeles.csv](manualis_teszteles/input/karakterkezeles.csv) <- `[karakterkezeles.csv](manualis_teszteles/input/karakterkezeles.csv)`